python_terraform/__init__.py

```diff
@@ -387,6 +389,41 @@ def read_state_file(self, file_path=None):
 
         self.tfstate = Tfstate.load_file(file_path)
 
+    def set_workspace(self, workspace):
+        """
+        set workspace
+        :param workspace: the desired workspace.
+        :return: status
+        """
+
+        return self.cmd('workspace select ' + workspace)  
+
```

Comment:
use 
```
self.cmd('workspace', 'select', workspace)
```

Reply:
Made the change and ensured it built and all tests pass.